docs/1.getting-started/3.configuration.md

```diff
@@ -29,6 +29,24 @@ Every configuration option is described in the [Configuration Reference](/docs/a
 You don't have to use TypeScript to build an application with Nuxt. However, it is strongly recommended to use the `.ts` extension for the `nuxt.config` file. This way you can benefit from hints in your IDE to avoid typos and mistakes while editing your configuration.
 ::
 
+You can configure fully type , per-environment overrides in your nuxt.config
+
```

Comment:
cc: @danielroe Could we suggest that these overrides are applied based on the key matching the `NODE_ENV` environment variable at dev time and build time (if that's how it actually works) ? 

Maybe explicitly show an example of how a config that has both $production and $development keys will be resolved ?

Reply:
Thanks for suggestion i  accepted the changes now what to do ?